server/src/main/java/org/opensearch/cluster/metadata/MetadataCreateIndexService.java

```diff
@@ -1044,7 +1044,10 @@ public void validateIndexSettings(String indexName, final Settings settings, fin
         for (final String key : settings.keySet()) {
             final Setting<?> setting = indexScopedSettings.get(key);
             if (setting == null) {
-                assert indexScopedSettings.isPrivateSetting(key) : "expected [" + key + "] to be private but it was not";
+                //https://github.com/opensearch-project/OpenSearch/issues/1019
```

Comment:
I see a lot of "// see: [github link]" in the code, maybe we can just adopt that "convention"

The fact that there's no space after `//` is really hurting my OCD ;)

Reply:
Done